components/composited/Notification/NotificationItem.tsx

```diff
@@ -0,0 +1,38 @@
+import Link from 'next/link';
+import { Fragment } from 'react';
+import { Badge } from '../../generic/Badge/Badge';
+import { LinkButton } from '../../generic/LinkButton/LinkButton';
+import { Text } from '../../generic/Text/Text';
+import { getNotificationContents, NotificationType } from './getNotificationContents';
+
+type NotificationItemProps = {
+  notificationType: NotificationType;
+  communityName: string;
+  isNew?: boolean;
+};
+
+export const NotificationItem = ({
+  notificationType,
+  communityName,
+  isNew = false,
+}: NotificationItemProps) => {
+  const [name, descPart1, descPart2] = getNotificationContents(notificationType);
```

Comment:
Create function `getTitle`, and `getDescription` parts. I would probably do it in a different way, but we don't have much time to think.